src/uu/rm/src/rm.rs

```diff
@@ -326,6 +326,27 @@ pub fn remove(files: &[&OsStr], options: &Options) -> bool {
 fn handle_dir(path: &Path, options: &Options) -> bool {
     let mut had_err = false;
 
+    #[cfg(unix)]
+    let del = "/";
+    #[cfg(windows)]
+    let del = "\\";
```

Comment:
Are you looking for [`std::path::MAIN_SEPARATOR`](https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html)?

Reply:
Yes, thank you for that help. I'll go ahead and make that change and push, and also try to follow up on the other branch